src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import '../../style.css';
+
+export interface SliderProps extends React.HTMLAttributes<HTMLDivElement> {
+  /** The value of the Slider */
+  value?: string;
+  /** The upper limit of the Slider */
+  max?: number;
+  /** The lower limit of the Slider */
+  min?: number;
+  /** The precision of the Slider */
+  step?: number;
+  /** The tile of the Slider, for displaying to user */
+  title?: string;
+  /** The name of the Slider */
+  name?: string;
+  /** The id of the Slider */
+  id?: string;
+}
+
+const Slider = ({
+  max = 100,
+  min = 0,
+  value = 'NaNey',
```

Comment:
What is NaNey?

Reply:
random text, probably NAN plus stuff
I'll change it to something more normal